terraform/environment/region/lambda_synchronisation.tf

```diff
@@ -1,88 +1,86 @@
-locals {
```

Comment:
Are we re-using a lambda for the audit sql stuff?
Is any of this sync functionality still needed?

Reply:
Nope none of it. I was going to pull it out properly as a separate PR as there's quite a bit of it lying around. Just commented it out here as some of my changes would have involved updating it to get it working as as it's not used, it wasn't worth the effort.